src/HttpReply.cc

```diff
@@ -116,6 +116,14 @@ HttpReply::pack() const
     return mb;
 }
 
+HttpReply *
```

Comment:
The al->reply being set from this return result stores HttpReplyPointer. Please just return one of those instead of this unique_ptr dance with raw-pointers.

Reply:
Done at d8c6441.